tutorials/3d/spring_arm.rst

```diff
@@ -0,0 +1,117 @@
+:article_outdated: False
+
+.. _doc_spring_arm:
+
+Third-person camera with spring arm
+===================================
+
+.. video:: video/spring_arm_camera.webm
+   :alt: Camera attached to a spring arm colliding with walls
+   :autoplay:
+   :loop:
+   :muted:
```

Comment:
Why is this same video at the top *and* bottom of the page?

I would probably just put it at the bottom, since it shows off the final version.

Reply:
I had also put it at that top to show what the tutorial will end with, but just having it at the bottom is fine too 👍